src/styles/caniuse.css.js

```diff
@@ -4,11 +4,44 @@ const css = String.raw;
 // Prettier ignore only to keep code indented from level 0.
 // prettier-ignore
 export default css`
+
 .caniuse-stats {
   display: flex;
-  flex-wrap: wrap;
+  column-gap: 1cm;
```

Comment:
Maybe use em/rem?

Reply:
Sure, I'll find an equivalent. 1cm looks good to base off.